recipes/apache-solr/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set name = "apache-solr" %}
+{% set version = "9.7.0" %}
+package:
+  name: {{ name | lower }}
+  version: {{ version }}  # Use the specific version you want to package
+
+source:
+  url: https://downloads.apache.org/solr/solr/{{ version }}/solr-{{ version }}.tgz
+  sha256: 38548b86fa4e3c87883875952da124bf7d742cb8f7b25d37a1176833588e8552
+
+build:
+  number: 0
+  noarch: generic
+
+requirements:
+  run:
+    - openjdk >=11  # Solr requires Java to run
+
+test:
+  commands:
+    - solr start -e cloud -noprompt  # [unix]
+    - solr stop -all  # [unix]
+    - solr.cmd start -e cloud -noprompt  # [win]
```

Comment:
don't need selectors here: `sold.cmd` will be on `%PATH%`, so can just be `solr`